src/common/Set/Set.tsx

```diff
@@ -60,18 +61,18 @@ export const Set: React.FC<SetProps> = ({ info }) => {
 
     if (info.filterBy === "happening_now") {
       // this is dumb af, i know
-      currArr = currArr.filter(
-        (item) =>
+      // yes it is, please replace "a" and "b" with descriptive names
+      currArr = currArr.filter((item) => {
+        const startTimeDate = new Date(item.event.startTime);
+        const durationMinutes = item.event.duration;
+        const a =
           currentTime.getTime() >
-            new Date(item.event.startTime).getTime() -
-              TIME_BEFORE_SHOW_BADGE_MS &&
+          new Date(startTimeDate).getTime() - TIME_BEFORE_SHOW_BADGE_MS;
+        const b =
```

Comment:
lmao please do name them better. Are we displaying "happening_now" badges some time before the events actually start?

Reply:
We had a "Starts in 15min" badge last year as well as a "Live" badge if it was ongoing. It would be nice to keep that functional

Reply:
I'll try and confirm if the Pill and "Live" badges are still working with some test events. Do we have a testing library for this project?